config.ru

```diff
@@ -0,0 +1,7 @@
+require "bundler/setup"
+
+require_relative "./store"
+
+use Rack::MethodOverride
```

Comment:
I read that it's necessary if I want use DELETE or PUT in sinatra, when I wanted to make a delete button.